userdocs/creating-a-codelist-from-a-csv-file.md

```diff
@@ -5,15 +5,22 @@ As well as creating a codelist from scratch, you can create one by uploading a C
 From the _My codelists_ page, click _Create a codelist_.
 
 Choose a name for the codelist, select a coding system from the dropdown,
-and choose a file to upload from your hard drive.  If you are a member of an organisation, you can also choose an owner for the codelist (your own account or an organisation account).
+and choose a file to upload from your hard drive.  If you wish to create an OPCS-4 codelist, please see the notes at the bottom of this page. If you are a member of an organisation, you can also choose an owner for the codelist (your own account or an organisation account).
```

Comment:
```suggestion
and choose a file to upload from your hard drive.

To create an OPCS-4 codelist, please see the notes at the bottom of this page.

If you are a member of an organisation, you can also choose an owner for the codelist (your own account or an organisation account).
```

This would be OK as I've suggested. It's actually not bad as written; just I think having separate paragraphs for each sentence helps make the individual points stand out.

Off-hand, I'm not sure it's possible to link to a heading in the Markdown implementation used here :thought_balloon:

It would be even better to link the notes:

`To create an OPCS-4 codelist, see [the explanation below](insert heading here).`

The other alternative might be to split the OPCS-4 codelist guidance into another Markdown file, and then just link that page.